common/djangoapps/course_modes/views.py

```diff
@@ -140,7 +142,23 @@ def get(self, request, course_id, error=None):
             in CourseMode.modes_for_course(course_key, only_selectable=False)
         )
         course_id = text_type(course_key)
+
+        program_urls = {}
+        bundles_on_track_selection = WaffleFlag(WaffleFlagNamespace(name=u'experiments'), u'bundles_on_track_selection')
```

Comment:
I'm assuming this is too expensive to compute so we can't include it in the user-metadata blob we are dumping on every page?

Reply:
I'd rather not add it to every page unless we know we're going to need it, but we should be able to add if we do

Reply:
Makes sense